server/main.lua

```diff
@@ -66,6 +66,10 @@ end)
 CreateThread(function()
     while true do
         local housePlants = MySQL.query.await('SELECT * FROM house_plants', {})
+        local stages = {
```

Comment:
Idk if it was the best way to store the stages. If someone want to add it to a config file or somewhere more desirable don't hesitate